auth/token.go

```diff
@@ -29,8 +30,8 @@ type Token interface {
 	Expiration() time.Time                                // Expiration returns "exp" claim, if it doesn't exist empty string is returned
 	IsExpired() bool                                      // IsExpired returns true, if 'exp' claim + leeway time of 1 minute is before current time
 	IssuedAt() time.Time                                  // IssuedAt returns "iat" claim, if it doesn't exist empty string is returned
-	Issuer() string                                       // Issuer returns "iss" claim, if it doesn't exist empty string is returned
-	IasIssuer() string                                    // IasIssuer returns "ias_iss" claim, if it doesn't exist empty string is returned
+	CustomIssuer() string                                 // CustomIssuer returns "iss" claim if it is a custom domain ("ias_iss" claim available), if it doesn't exist empty string is returned
+	Issuer() string                                       // Issuer returns "ias_iss" (SAP domain, only set if custom domains are used) claim, if it doesn't exist the standard "iss" claim is returned
```

Comment:
the comment is hard to understand - maybe sth like
// returns "ias_iss" claim in case of custom domains, otherwise "iss" claim is returned

Reply:
I thought we are renaming the former `IasIssuer` method because consumers would'nt understand what it is? 
I rephrased it a bit but kept the explanation of the ias_iss claim.